lib/share_progress/variation.rb

```diff
@@ -0,0 +1,92 @@
+require 'share_progress'
+require 'share_progress/client'
+
+module ShareProgress
```

Comment:
this is also going to want a 'button_id' attr_accessor which will be useful when the save method is implemented